sql/updates/latest-dev.sql

```diff
@@ -373,16 +373,42 @@ ALTER EXTENSION timescaledb
 
 DROP TABLE _timescaledb_internal.job_errors;
 
-UPDATE
-  _timescaledb_config.bgw_job
-SET
-  application_name = 'Job History Log Retention Policy [2]',
-  proc_schema = '_timescaledb_functions',
-  proc_name = 'policy_job_stat_history_retention',
-  check_schema = '_timescaledb_functions',
-  check_name = 'policy_job_stat_history_retention_check'
-WHERE
-  id = 2;
+DELETE FROM _timescaledb_config.bgw_job WHERE id = 2;
```

Comment:
What about just turn `scheduled=false` instead?? So the old job errors data (migrated) will not have an orphan record.

Reply:
done